core/arch/arm/kernel/entry_a32.S

```diff
@@ -823,9 +829,9 @@ LOCAL_FUNC enable_mmu , : , .identity_map
 	bx	lr
 END_FUNC enable_mmu
 
-LOCAL_DATA stack_tmp_export_rel , :
-	.word	stack_tmp_export - stack_tmp_export_rel
-END_DATA stack_tmp_export_rel
+LOCAL_DATA stack_tmp_rel , :
+	.word	stack_tmp - stack_tmp_rel
```

Comment:
I'm not sure but it seems that making this `stack_tmp - stack_tmp_rel - STACK_TMP_GUARD` should allow us to skip the entire `#if (STACK_TMP_GUARD != 0)` above.

Reply:
It does work. Updated.

Reply:
Great, please do the same in the _a64.S.

Reply:
Done.